Server/Program.cs

```diff
@@ -1,12 +1,41 @@
 ﻿using System;
+using System.Diagnostics;
 
 namespace Server
 {
+    using Dashboard;
+    using Dashboard.Client.SessionManagement;
```

Comment:
Why is Server using Dashboard.Client.SessionManagement ?

Reply:
because EndMeet() method is in the ClientSessionManagement class.

Reply:
Yes, the UX server doesn't need the EndMeet() method.  EndMeet() method is for client to end the meeting.